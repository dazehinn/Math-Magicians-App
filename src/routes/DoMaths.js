import Calculator from '../components/calculator';

const DoMaths = () => (
  <div>
    <h2>Let&apos;s do some maths</h2>
    <Calculator />
  </div>
);
export default DoMaths;
