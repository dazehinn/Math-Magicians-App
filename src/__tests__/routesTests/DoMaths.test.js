import renderer from 'react-test-renderer';
import DoMaths from '../../routes/DoMaths';

describe('Home Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<DoMaths />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
