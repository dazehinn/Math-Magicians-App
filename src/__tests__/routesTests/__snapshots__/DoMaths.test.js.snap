// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Tree 1`] = `
<div>
  <h2>
    Let's do some maths
  </h2>
  <div
    className="calculatorContainer"
  >
    <div
      className="outputDisplay"
    >
      0
    </div>
    <div
      className="operatorSection"
    >
      <button
        className="operator"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="operator"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="operator"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="operator"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="operator"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
    <div
      className="operandSection"
    >
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="digit zero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="digit"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
    </div>
  </div>
</div>
`;
